packages/connections/src/components/connection-list/connection-list.tsx

```diff
@@ -73,6 +76,28 @@ function ConnectionList({
   createNewConnection: () => void;
   setActiveConnectionId: (connectionId?: string) => void;
 }): React.ReactElement {
+  const favoriteConnections = connections
+    .filter(
+      (connectionInfo): connectionInfo is ConnectionInfoFavorite =>
```

Comment:
First time using this in typescript - it's a way to pass the filtered type down the chain.